docs/conf.py

```diff
@@ -96,15 +96,25 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'custom'
+html_theme = 'sphinx_scylladb_theme'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
-#html_theme_options = {}
+# Theme options are theme-specific and customize the look and feel of a theme
```

Comment:
this comment is duplicated... for no need...

Reply:
good catch! fixed 😄